examples/nlp/glue_benchmark/glue_benchmark_config.yaml

```diff
@@ -40,14 +40,23 @@ model:
     num_samples: -1
     batch_size: 32
 
+
+  tokenizer:
+      tokenizer_name: ${model.language_model.pretrained_model_name} # or sentencepiece
+      vocab_file: null # path to vocab file
+      data_file: null # used to build sentencepiece
+      sample_size: null # for sentencepiece
+      vocab_size: null # vocab size for building sentence piece
+      do_lower_case: null # only used when tokenizer is build with vocab file or with sentencepiece  
+      tokenizer_model: null # only used if tokenizer is sentencepiece
+      special_tokens: null # only necessary for adding transformer/bert-specific special tokens to tokenizer if the tokenizer does not already have these inherently.
```

Comment:
let's discuss in the upcoming meeging if we can reduce/remove/regroup these sentencepiece only related args

Reply:
done, agreed on removing vocab_size, sample_size, data_file if everything works as expected